components/select_team/index.ts

```diff
@@ -2,28 +2,37 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
-import {bindActionCreators, Dispatch} from 'redux';
+import {bindActionCreators, Dispatch, compose} from 'redux';
 import {withRouter} from 'react-router-dom';
 
 import {getTeams} from 'mattermost-redux/actions/teams';
 import {loadRolesIfNeeded} from 'mattermost-redux/actions/roles';
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
 import {Permissions} from 'mattermost-redux/constants';
 import {haveISystemPermission} from 'mattermost-redux/selectors/entities/roles';
 import {getSortedListableTeams, getTeamMemberships} from 'mattermost-redux/selectors/entities/teams';
 import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {cloudFreeEnabled} from 'mattermost-redux/selectors/entities/preferences';
+
+import withUseGetUsageDelta from 'components/common/hocs/cloud/with_use_get_usage_deltas';
 
 import {GlobalState} from 'types/store';
 
 import {addUserToTeam} from 'actions/team_actions';
 import {isGuest} from 'mattermost-redux/utils/user_utils';
 
+import {isCloudLicense} from 'utils/license_utils';
+
 import SelectTeam from './select_team';
 
 function mapStateToProps(state: GlobalState) {
     const config = getConfig(state);
     const currentUser = getCurrentUser(state);
     const myTeamMemberships = Object.values(getTeamMemberships(state));
+    const license = getLicense(state);
+
+    const isCloud = isCloudLicense(license);
+    const isCloudFreeEnabled = isCloud && cloudFreeEnabled(state);
```

Comment:
After launch we won't be using the freemium feature flag. We can remove this check and just check for cloud in the license

Reply:
Removed.